Assets/Scripts/Model/Director.cs

```diff
@@ -136,7 +206,7 @@ public void NextState(WorldState state)
                 {
                     if (fabric == null)
                         continue;
-                    Materials += fabric.DoProcessing();
+                    Product += fabric.DoProcessing();
```

Comment:
Are `Product` and `Material` the same things?

Reply:
No, materials become products.